src/bundle/Resources/public/scss/_popup-menu.scss

```diff
@@ -1,53 +1,78 @@
 .ibexa-popup-menu {
-    padding: calculateRem(8px);
+    display: flex;
+    flex-direction: column;
+    gap: calculateRem(1px);
+    padding: calculateRem(8px) 0;
     background: $ibexa-color-white;
     border: calculateRem(1px) solid $ibexa-color-light;
     border-radius: $ibexa-border-radius;
     box-shadow: calculateRem(4px) calculateRem(22px) calculateRem(67px) 0 rgba($ibexa-color-info, 0.2);
     position: absolute;
-    z-index: 10;
+    z-index: 1060;
 
-    &--hidden {
+    &__items-list {
+        padding-left: 0;
+    }
+
+    &__subitems-arrow {
         display: none;
+        position: absolute;
+        right: calculateRem(8px);
+        top: calc(50% - calculateRem(16px) / 2);
```

Comment:
I think that does not work and you have to use `${calculateRem(xxx)` inside calc

Reply:
it works